std_msgs/Header header
uint32 casualty_id
sensor_msgs/NavSatFix location
sensor_msgs/Image image
std_msgs/Time time_ago
